building/build-debs/homeworld-keysystem/src/keyserver/authorities/static.go

```diff
@@ -0,0 +1,25 @@
+package authorities
+
+import (
+	"errors"
+)
+
+type StaticAuthority struct {
```

Comment:
Could you please explain what a static authority is and how it is used?

Reply:
It's used for the kubernetes serviceaccount key, which needs to be shared across all of the different master nodes, so each one is given a copy of the master key instead of getting a certificate.